src/tasks/test/AltClickMinimapTest.py

```
@@ -0,0 +1,77 @@
+        self.name = "Alt点击小地图测试"
+        self.group_name = "工具与调试"
+        self.group_icon = FluentIcon.DEVELOPER_TOOLS
+        self.description = "验证 alt+点击小地图中心能否替代 send_key('m') 打开地图（后台输入可行性）"
+        self.visible = self.debug
+
+        self.default_config = {
+            "测试次数": 3,
+            "测试前前置游戏": False,
+        }
+        self.config_description = {
+            "测试次数": "重复测试次数，每次先关闭地图再重新打开",
```

Ruff (0.16.2)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)